06_puppy/mohitnag/types.go

```diff
@@ -0,0 +1,27 @@
+package main
+
+import "sync"
+
+// Puppy defines the data structure corresponding to a Puppy
+type Puppy struct {
+	ID     uint32
+	Breed  string
+	Colour string
+	Value  string
+}
+
+// MemStore stores Puppy details with Puppy Id as Key and Puppy  as value
+type MemStore map[uint32]Puppy
```

Comment:
IMO this belongs in the memstore.go file.

Reply:
I think all the types are under one file i.e 'types.go' , which could be more readable for some. However not sure which is the best way.